tests/framework/cli/test_starters.py

```diff
@@ -291,12 +299,12 @@ def test_parse_add_ons_invalid_range(input, capsys):
 
 @pytest.mark.parametrize(
     "input,first_invalid",
-    [("0,3,5", "0"), ("1,3,6", "6"), ("0-4", "0"), ("3-6", "6")],
+    [("0,3,5", "0"), ("1,3,7", "7"), ("0-4", "0"), ("3-7", "7")],
```

Comment:
Why not just make the invalid number "100"? That way we don't have to keep bumping this with every new add on.

Reply:
I believe its because the `_validate_selection` function checks each number in the list of selected add-ons and on the first invalid selection it raises an error which for now is 7, the test will always have to know the first invalid selection.